core/jazz_logs/config/test-config.json

```diff
@@ -2,53 +2,79 @@
   "KIBANA_URL": "{inst_kibana_search_hostname}",
   "DEFAULT_SIZE": 1000,
   "DEFAULT_TIME_IN_DAYS": 7,
-  "VALID_ENVIRONMENTS": ["prod", "stg", "dev", "test"],
-  "VALID_CATEGORIES": ["api", "function", "sls-app"],
-  "VALID_ASSET_TYPES": ["lambda", "apigateway"],
-  "ASSET_INDEX_MAP":[{
-    "asset_type": "lambda",
-    "es_index": ["applicationlogs"]
-  }, {
-    "asset_type": "apigateway",
-    "es_index": ["apilogs"]
-  }],
-  "VALID_LOGTYPES": ["warn", "error", "info", "verbose", "debug"],
-  "ENV_PREFIX": "bA11r00m-dAnCe",
-  "LOG_LEVELS": [{
+  "VALID_ENVIRONMENTS": [
+    "prod",
+    "stg",
+    "dev",
+    "test"
+  ],
+  "VALID_CATEGORIES": [
+    "api",
+    "function",
+    "sls-app"
+  ],
+  "VALID_ASSET_TYPES": [
+    "lambda",
+    "apigateway"
+  ],
+  "ASSET_INDEX_MAP": [
+    {
+      "asset_type": "lambda",
+      "es_index": "applicationlogs"
+    },
+    {
+      "asset_type": "apigateway",
+      "es_index": "apilogs"
+    }
+  ],
+  "VALID_LOGTYPES": [
+    "WARN",
+    "ERROR",
+    "INFO",
+    "VERBOSE",
+    "DEBUG",
+    "FATAL",
+    "TRACE",
+    "OFF",
+    "ALL"
+  ],
+  "ENV_PREFIX": "{env-prefix}",
+  "LOG_LEVELS": [
+    {
       "Level": 0,
-      "Type": "off"
+      "Type": "OFF"
     },
     {
       "Level": 1,
-      "Type": "fatal"
+      "Type": "FATAL"
```

Comment:
Why this switch?

Reply:
Everything seems to have changed from lowercase to uppercase?